### ISSUE TITLE
README example failed

### ISSUE THREAD

**Issue by dannypsnl** [second-state/wasmedge_plugin_rust_sdk#1]
Run

```
cd examples/plugin/stateful_plugin
cargo build --release
```

get

```
   Compiling wasmedge_plugin_sdk v0.1.0
error[E0599]: no method named `cast_mut` found for raw pointer `*const WasmEdge_ModuleInstanceContext` in the current scope
  --> /home/dannypsnl/.cargo/registry/src/github.com-1ecc6299db9ec823/wasmedge_plugin_sdk-0.1.0/src/sdk/module.rs:60:52
   |
60 |             inst: InnerInstance::from_raw(inst_ctx.cast_mut()),
   |                                                    ^^^^^^^^ help: there is an associated function with a similar name: `as_mut`
   |
   = note: try using `<*const T>::as_ref()` to get a reference to the type behind the pointer: https://doc.rust-lang.org/std/primitive.pointer.html#method.as_ref
   = note: using `<*const T>::as_ref()` on a pointer which is unaligned or points to invalid or uninitialized memory is undefined behavior
```

### wasmedge version

```
$ wasmedgec --version
wasmedgec version 0.11.2
```

**Comment by L-jasmine** [comment]
what's rust version?

**Comment by dannypsnl** [comment]
```
rustc 1.61.0
```

Which version should I use?

**Comment by L-jasmine** [comment]
my rust is 1.67.1.
I will fix this to support low version.

**Comment by dannypsnl** [comment]
Solved after using 1.67.1, so I guess just need to add minimal rust requirement.

**Comment by hydai** [comment]
Hi @L-jasmine 
Just a question about the rust ecosystem. Will users keep the old version on their platforms? Or will they constantly update the rust toolchain to the latest 2 or 3 releases?
